lib/uplink-gomobile/build.sh

```diff
@@ -11,9 +11,13 @@ then
       exit 1
 fi
 
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
 # setup tmpdir for testfiles and cleanup
 TMP=$(mktemp -d -t tmp.XXXXXXXXXX)
 cleanup(){
+      # cleanup pkg/mod directory
+      go clean -modcache
```

Comment:
Wait, why are you deleting the mod cache? Currently the mod cache is shared between different Jenkins build runs, this would make everything redownload the dependencies.

Reply:
ah true, it was necessary before I linked `pkg` because I was not able to delete `$TMP` completely but now I think I can remove it.